internal/client.go

```diff
@@ -169,8 +167,16 @@ func NewClient(opts ...ClientOption) (*Client, error) {
 	c := &Client{
 		httpClient: http.DefaultClient,
 	}
+
+	certPool, err := x509.SystemCertPool()
+	if err != nil {
+		return nil, fmt.Errorf("failed to get system cert pool: %s", err)
+	}
+	c.certpool = certPool
+
 	c.httpClient.Transport = http.DefaultTransport
 	c.httpClient.Transport.(*http.Transport).TLSClientConfig = &tls.Config{}
+	c.httpClient.Transport.(*http.Transport).TLSClientConfig.RootCAs = c.certpool
```

Comment:
Doesn't this modify default transport?

Reply:
Good catch; using cleanhttp to generate a new client.